include/cpuinfo_x86.h

```diff
@@ -140,6 +140,7 @@ typedef enum {
   INTEL_BDW,           // BROADWELL
   INTEL_SKL,           // SKYLAKE
   INTEL_ATOM_GMT,      // GOLDMONT
+  INTEL_ATOM_GMT_P,    // GOLDMONT+
```

Comment:
line 172 below we've used `AMD_ZEN_PLUS` so I'm thinking if we should use `INTEL_ATOM_GMT_PLUS` instead :thinking:

Reply:
ok